src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceBaseSourceConfig.java

```diff
@@ -75,6 +76,11 @@
   @Macro
   private String offset;
 
+  @Name(SalesforceSourceConstants.PROPERTY_OPERATION)
+  @Description("If set to query, the query result will only return current rows. If set to queryAll, " +
+          "all records, including deletes will be sourced")
+  private String operation;
```

Comment:
should this param be macro enabled?
Also, seems like we should support null operation for backward compatibility, so let's make it nullable and default to 'query' when operation is not specified

Reply:
I have this set as a radio button, defaulted to "query"...does this work?

Reply:
That works when the user is building a pipeline in pipeline studio. 
But we also need to support the case where a pipeline JSON with an older version of salesforce plugin is imported. 
The old pipeline json won't have 'operation' config.

Reply:
got it. I've made it nullable